### DISCUSSION TITLE
Vercel deploy build can't find husky

### DISCUSSION THREAD

**Discussion by GraysonQuartech** [mckaywrigley/chatbot-ui#1509] [Help]
i follow the instructions and then try to run verel --prod in the vs code project and it starts running and then hangs up on 
'husky - git command not found, skipping install'

anyone else getting this?   tried some of the solutions here https://github.com/typicode/husky/issues/945  with no luck

**Comment by divyajangid2496** [comment]
Facing same issue. Were you able to solve this?
